plugins/basic/python/tk_maya_basic/plugin_logic.py

```diff
@@ -270,5 +267,12 @@ def _jump_to_website():
     """
     Jumps to the Shotgun website in the defaul web browser.
     """
+    QtGui.QDesktopServices.openUrl(QtCore.QUrl("https://www.shotgunsoftware.com"))
+
+
+def _jump_to_signup():
+    """
+    Jumps to the Shotgun website in the defaul web browser.
```

Comment:
default (line 268 too)
The Shotgun sign up website?  This one's different than the one above ;)